src/shared/components/create/NoteCreator.tsx

```diff
@@ -207,16 +212,7 @@ function NoteCreator({handleClose, quotedEvent, repliedEvent}: NoteCreatorProps)
     event.kind = 1
     event.content = noteContent
     event.tags = []
-    if (repliedEvent) {
-      event.tags = [
-        ["p", repliedEvent.pubkey],
-        ["e", repliedEvent.id, "", "reply", repliedEvent.pubkey],
-      ]
-    }
-    if (quotedEvent) {
-      event.tags.push(["e", quotedEvent.id, "", "mention", quotedEvent.pubkey])
-    }
-    addPTags(event, repliedEvent, quotedEvent)
+    addTags(event, repliedEvent, quotedEvent)
```

Comment:
Renamed `addPTags` to `addTags` because it was doing more than just adding p-tags.